test/extended/node_tuning/node_tuning_utils.go

```diff
@@ -0,0 +1,61 @@
+package node_tuning
+
+import (
+	"regexp"
+	"strings"
+
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var oc = exutil.NewCLIWithoutNamespace("nto")
+
+// IsSNOCluster will check if OCP is a single node cluster
+func isSNOCluster() bool {
+
+	topologyTypeStdOut, err := oc.AsAdmin().WithoutNamespace().Run("get").Args("infrastructure", "-ojsonpath={.items[*].status.infrastructureTopology}").Output()
+	o.Expect(err).NotTo(o.HaveOccurred())
+	o.Expect(topologyTypeStdOut).NotTo(o.BeEmpty())
+	topologyType := strings.ToLower(topologyTypeStdOut)
+	return topologyType == "singlereplica"
+}
+
+func getFirstMasterNode() string {
+	var firstMasterNodeName string
+	masterNodeNames, err := oc.AsAdmin().WithoutNamespace().Run("get").Args("nodes", "-l node-role.kubernetes.io/control-plane=", `-ojsonpath='{.items[*].status.addresses[?(@.type=="Hostname")].address}'`).Output()
```

Comment:
This seems a bit of a hack.  Would it be possible to use a native nodeClient for this?  For example: `oc.KubeClient().CoreV1().Nodes()` . Example: https://github.com/openshift/origin/blob/25201454c47050ee3179a3dae894cafd72162ed2/test/extended/etcd/vertical_scaling.go#L46

And them somthing like https://github.com/openshift/origin/blob/25201454c47050ee3179a3dae894cafd72162ed2/test/extended/etcd/helpers/helpers.go#L271